### ISSUE TITLE
Replacenear command broken -- Too many parameters, null.

### ISSUE THREAD

**Issue by mibby** [IntellectualSites/FastAsyncWorldEdit-1.13#52]
Paper dev 525 (Spigot 1.13.2)
FAWE 1.13 compiled as of commit https://github.com/IntellectualSites/FastAsyncWorldEdit-1.13/commit/24fbc86cdd79b8ce13359342bc422ce1fb25a0d2

``/replacenear 10 pink_concrete dark_prismarine``

Outputs;
```
Too many parameters! Unused parameter: dark_prismarine
null
```

No errors in console.

**Comment by mibby** [comment]
Likewise, also got a report from someone of ```/replace``` not working properly.

> Seems like the //replace function is partly broken. I selected an area, I wanted to change the sand into grass blocks, did the selection, did a //replace sand grass_block but it replaced all the blocks with sand instead, as if it ignored the 2nd parameter.

> For the //replace, I even tried with a diffent block, or a "banned" block (like grass) and it doesn't give an error, still replaces all to sand.

**Comment by IronApollo** [comment]
Fixed in 3ffb5ff00d2ec3a7dc1a9a35546aa197832a9be1. Closed.
